### PR TITLE
Changes to incorporate coordinate transformations

### PR DESCRIPTION
**Pull request by spxiwh**
Hi Patricia,

I'll send an email explaining what I'm doing here.

### CODE REVIEW

File: pycbc/waveform/nr_waveform.py [hunk 1]
```diff
@@ -43,12 +43,100 @@ def get_data_from_h5_file(filepointer, time_series, key_name):
     knots = filepointer[key_name]['knots'][:]
     data = filepointer[key_name]['data'][:]
     # Check time_series is valid
-    assert(knots[0] <= time_series[0])
-    assert(knots[-1] >= time_series[-1])
```
Comment (spxiwh) [inline pycbc/waveform/nr_waveform.py]: This catches an existing float [over,under]flow error where knots[0] becomes float precision less than time_series[0]

File: pycbc/waveform/nr_waveform.py [hunk 2]
```diff
@@ -172,17 +263,24 @@ def get_param(value):
...
-            curr_ylm = lal.SpinWeightedSphericalHarmonic(theta, phi, -2, l, m)
-            hp += curr_h_real * curr_ylm.real + curr_h_imag * curr_ylm.imag
-            # FIXME: No idea whether these should be minus or plus, guessing
-            #        minus for now. Only affects some polarization phase
-            hc += + curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
+            curr_ylm = lal.SpinWeightedSphericalHarmonic(theta, psi, -2, l, m)
+            # Here is what 0709.0093 defines h_+ and h_x as. This defines the
+            # NR wave frame
+            curr_hp = curr_h_real * curr_ylm.real - curr_h_imag * curr_ylm.imag
+            curr_hc = -curr_h_real*curr_ylm.imag - curr_h_imag * curr_ylm.real
```
Comment (spxiwh) [inline pycbc/waveform/nr_waveform.py]: Note that the convention now matches the NR document.

File: pycbc/waveform/nr_waveform.py (hunk 1 above)
Comment (PatriciaSchmidt) [inline pycbc/waveform/nr_waveform.py]: Ok, cool. Looks good to me.

File: pycbc/waveform/nr_waveform.py (hunk 2 above)
Comment (PatriciaSchmidt) [inline pycbc/waveform/nr_waveform.py]: Which NR document? The data format one?

**Pull request merged**